### ISSUE TITLE
module 'astropy.timeseries.periodograms.lombscargle_multiband.implementations.main' has no attribute '_is_regular'

### ISSUE THREAD

**Issue by darthoctopus** [lightkurve/lightkurve#1350]
#### Problem description

As of recent versions of astropy, `LightCurve.to_periodogram` will fail  when calling `implementations.main._is_regular` with an AttributeError. This is because `implementations` in `astropy.timeseries` is now that of `astropy.timeseries.periodograms.lombscargle_multiband` rather than `lombscargle`, which does not have this helper function.

I changed line 23 of `periodogram.py` to

```
from astropy.timeseries.periodograms.lombscargle import implementations
```

and it seems to work for me now.

#### Example

Any call to `LightCurve.to_periodogram`

#### Expected behavior

return Periodogram object correctly

#### Environment

-  platform (e.g. Linux, OSX, Windows): Linux
-  lightkurve version (e.g. 1.0b6): 2.4.0 (but I imagine earlier versions will fail too)
-  installation method (e.g. pip, conda, source): pip

**Comment by keyuxing** [comment]
See https://github.com/lightkurve/lightkurve/issues/1347#issuecomment-1621071068

**Comment by Nschanche** [comment]
Hi @darthoctopus, as @keyuxing noted, this issue has been addressed and merged into lightkurve in #1342. The updated lightkurve package has not been released but you can install the dev version or use an older astropy version as suggested in #1347.

I am going to close this issue now, but if you have further problems or comments, you can open it back up. Thanks for submitting your issue!
